File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map <int, int> freq;
        for(int i=0; i<nums.size(); i++){
            freq[nums[i]]++;
        }
        for(auto e : freq){
            if(e.second > 1)
                return true;
        }
        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        # for num in nums:
        #     if nums.count(num) > 1:
        #         return True
        # return False
        nums.sort()
        i = 1
        while i < len(nums):
            if nums[i-1] == nums[i]:
                return True
            i += 1
        return False
        